### ISSUE TITLE
Handling empty boolean query params

### ISSUE THREAD

**Issue by juuuuuu** [apideck-libraries/php-sdk#40]
Hello

I started using this SDK today and encountered a problem.

As of today, I'm stuck with an old version of Guzzle PSR7 package (`1.9.1`).
This SDK is allowing this version (`guzzlehttp/psr7: ^1.7 || ^2.0`).
But in this version, boolean query params are not well handled.

This [has been fixed](https://github.com/guzzle/psr7/pull/391/files) in version `2.0.0`, but as previously said I'm stuck at the moment with this version by another dependency.

So, I was wondering if you could handle the following error in the case the query params look like that `[...]?raw=&[...]`?

So, passing `$raw`
- `false` renders `[...]?raw=&[...]`
- `true` renders `[...]?raw=1&[...]`

**Current Response**
```json
{
  "status_code": 400,
  "error": "Bad Request",
  "type_name": "RequestValidationError",
  "message": "Request Validation Error",
  "detail": {
    "errors": [
      {
        "message": "'raw' property type must be boolean",
        "path": "request.query.raw",
        "context": {
          "errorType": "type"
        }
      }
    ]
  },
  "ref": "https://developers.apideck.com/errors#requestvalidationerror"
}
```

**Current workaround**

Using `$raw` set to  `true` :sweat_smile:

**Comment by brahimmouhamou** [comment]
@juuuuuu this should be fixed in `v2.5.1`, let me know if this works for you.

**Comment by juuuuuu** [comment]
I can confirm it's working!
Thank you :)
